hnn_core/gui/gui.py

```diff
@@ -1271,12 +1282,11 @@ def load_drive_and_connectivity(params, log_out, drives_out,
                                 drive_widgets, drive_boxes, connectivity_out,
                                 connectivity_textfields, tstop, layout):
     """Add drive and connectivity ipywidgets from params."""
-    log_out.clear_output()
```

Comment:
The logging output should never be cleared ... otherwise it has the potential to confuse users by erasing important messages that were printed out